src/DynamoCore/Graph/Workspaces/UndoRedo.cs

```diff
@@ -200,6 +200,7 @@ internal void RecordAndDeleteModels(List<ModelBase> models)
             // to are deleted. We will have to delete the connectors first
             // before
 
+            using (BeginDelayedGraphExecution())// Delayed execution
```

Comment:
improves performance of the `RecordAndDeleteModels` operation

Reply:
I see there's just one RequestRun at the end of the `using(BeginActionGroup())`. Isn't this `using` going to do the same thing?

Reply:
Only the `RequestRun` from the DelayedGraphExecution destructor should run (because that will be the last one invoked).
Any other RequestRun before that will not do anything. It is safe to leave it as is...no need to remove all RequestRun calls within this code block...

Reply:
Yes we could

Reply:
Can you make the change.

Reply:
Done